catroid/build.gradle

```diff
@@ -140,6 +140,11 @@ android {
         versionName "0.9.66"
         println "VersionCode is $versionCode"
         println "VersionName is $versionName"
+        buildConfigField "String", "GIT_COMMIT_INFO", "\"${getGitCommitInfo()}\""
```

Comment:
what do you need the GIT_COMMIT_INFO for?

Reply:
seems to be there before, weird diff